pkg/web/templates/partials/transaction/transaction_accept.html

```diff
@@ -1,33 +1,50 @@
 <section class="my-4">
   <section>
     <div class="mb-4 md:mb-0">
-      <span class="font-semibold">Transaction ID: {{ .Transaction.Txid }}</span>
+      <span class="font-semibold">Transaction ID: {{ .Pending.EnvelopeId }}</span>
     </div>
   </section>
-  <form id="transaction-accept" hx-post="/v1/transactions/{{ .Transaction.Txid }}/send" hx-swap="none"
+  <form id="transaction-accept" hx-post="/v1/transactions/{{ .Pending.EnvelopeId }}/send" hx-swap="none"
     method="post">
     <section class="py-2 border-t border-t-black">
       <h2 class="my-4 font-bold">1. TRANSACTION DETAILS</h2>
       <div class="grid gap-6 my-4 md:grid-cols-2">
+        <div>
+          <label for="txid" class="label-style">Txid</label>
```

Comment:
This field was added after noting it wasn't originally included.

Reply:
good catch!